app-games/src/main/java/com/aptoide/android/aptoidegames/feature_oos/OutOfSpaceAppItem.kt

```diff
@@ -93,30 +91,24 @@ fun AppItem(
     ) {
       Text(
         text = name,
-        style = AppTheme.typography.descriptionGames,
+        style = AGTypography.DescriptionGames,
         modifier = Modifier.padding(bottom = 8.dp),
         overflow = TextOverflow.Ellipsis,
         maxLines = 2,
-        color = AppTheme.colors.outOfSpaceDialogAppNameColor
+        color = Palette.White
       )
       Text(
         text = TextFormatter.formatBytes(appSize),
-        style = AppTheme.typography.inputs_S,
+        style = AGTypography.InputsS,
         overflow = TextOverflow.Ellipsis,
         maxLines = 1,
-        color = AppTheme.colors.outOfSpaceDialogAppSizeColor
+        color = Palette.GreyLight
       )
     }
 
-    AppGamesOutlinedButton(
-      title = "Uninstall",
+    SecondarySmallOutlinedButton(
       onClick = action,
-      enabled = true,
-      style = Default(fillWidth = false),
-      modifier = Modifier
-        .wrapContentWidth()
-        .height(32.dp)
+      title = "Uninstall",
```

Comment:
Not related to PR but is this string supposed to be hardcoded ?

Reply:
Fixed